pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -207,23 +207,23 @@ in {
   # to be adapted
   zfsStable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.14";
-    latestCompatibleLinuxPackages = linuxPackages_5_13;
+    kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.15";
+    latestCompatibleLinuxPackages = linuxPackages_5_14;
 
     # this package should point to the latest release.
-    version = "2.1.0";
+    version = "2.1.1";
 
-    sha256 = "sha256-YdY4SStXZGBBdAHdM3R/unco7ztxI3s0/buPSNSeh5o=";
+    sha256 = "sha256-UUuJa5w/GsEvsgH/BnXFsP/dsOt9wwmPqKzDxLPrhiY=";
   };
 
   zfsUnstable = common {
     # check the release notes for compatible kernels
     kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.15";
-    latestCompatibleLinuxPackages = linuxPackages_5_13;
+    latestCompatibleLinuxPackages = linuxPackages_5_14;
 
     # this package should point to a version / git revision compatible with the latest kernel release
-    version = "unstable-2021-08-30";
-    rev = "3b89d9518df2c7fd747e349873a3d4d498beb20e";
+    version = "2.1.1";
+    # rev = "0000000000000000000000000000000000000000";
```

Comment:
Oops?

Reply:
Not oops, there's no commit after the 2.1.1 release so might as well just make them the same?

I didn't want to get rid of the rev for whenever someone goes to update the unstable they can easily see it there.

Reply:
Oh, I'm an idiot. One moment.

Reply:
Like, `master` doesn't add support for 5.15 yet, so what's the point of having an unstable where you don't gain support for any new kernels?

Reply:
Sorry, I commented on the wrong line.